functional/durable-attestion-sanity-on-localhost/test.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+HTTP_SERVER_PORT=8080
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+        
+        # clean previous durable attestion artifacts stored on filesystem
+        rlRun "rm -rf /var/lib/keylime/da/CI_*"
+
+        # copy fake binary_bios_measurements into an alternative path (used to create refstate)
+        rlRun "cp binary_bios_measurements /var/tmp"
```

Comment:
I believe you are not changing a working directory so this copy is probably not necessary.

Reply:
Ignore my comment, it is necessary.